examples/nginx-centos7/s2i/bin/save-artifacts

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+#
+# S2I save-artifacts script for the 'nginx-centos7' image.
+# The save-artifacts script streams a tar archive to standard output.
+# The archive contains the files and folders you want to re-use in the next build.
+#
+# For more information see the documentation:
+#	https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md
+#
+tar cf - /usr/share/nginx/html/
```

Comment:
@coreydaley this doesn't make sense, please change it.  Either no command, or `echo artifact >/tmp/artifact; tar -C /tmp -c artifact` would be preferable.

@bparees fwiw (and fortunately), s2i appears to be perfectly happy with a zero length tar stream.